py3status/modules/volume_status.py

```diff
@@ -92,6 +92,11 @@
 from os import devnull, environ as os_environ
 from subprocess import check_output, call, CalledProcessError
 
+STRING_ERROR = 'invalid command `%s`'
+STRING_NOT_AVAILABLE = 'no available binary'
+STRING_NOT_INSTALLED = 'not installed'
+COMMAND_NOT_INSTALLED = 'command `%s` {}'.format(STRING_NOT_INSTALLED)
```

Comment:
`'command `%s` not installed'` is much better.

The reason is because At some point I'd like us to do translations and for that we cannot make such assumptions about how things are expressed.